README.md

```diff
@@ -1,4 +1,4 @@
-# Kubeflow Manifests
+# Kubeflow on IKS(IBM Cloud Kubernetes Service)
```

Comment:
@yhwang could we drop this change? I think it was most probably done from a quick copy paste :-)

Reply:
you're right. fixed